src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -6849,24 +6867,26 @@ private void Exec(Instruction instruction)
             }
         }
 
-        private void GC()
+        public List<StackValue> CollectRootPointers()
```

Comment:
Public seems rather broad for this. Is there a reason why this should be part of our API?